source/llvm/config.d

```diff
@@ -55,32 +56,36 @@ immutable LLVM_Version = asVersion(LLVM_VERSION_MAJOR, LLVM_VERSION_MINOR, LLVM_
 immutable LLVM_Targets = {
 	string[] targets;
 	mixin({
-		       static if (LLVM_Version >= asVersion(3, 9, 0)) {
-			return ["AArch64","AMDGPU","ARM","AVR","BPF","Hexagon","Lanai","MSP430","Mips","NVPTX","PowerPC","Sparc","SystemZ","WebAssembly","X86","XCore"];
-		} else static if (LLVM_Version >= asVersion(3, 8, 0)) {
-			return ["AArch64","AMDGPU","ARM","AVR","BPF","CppBackend","Hexagon","MSP430","Mips","NVPTX","PowerPC","Sparc","SystemZ","WebAssembly","X86","XCore"];
-		} else static if (LLVM_Version >= asVersion(3, 7, 0)) {
-			return ["AArch64","AMDGPU","ARM","BPF","CppBackend","Hexagon","MSP430","Mips","NVPTX","PowerPC","Sparc","SystemZ","WebAssembly","X86","XCore"];
-		} else static if (LLVM_Version >= asVersion(3, 6, 0)) {
-			return ["AArch64","ARM","CppBackend","Hexagon","MSP430","Mips","NVPTX","PowerPC","R600","Sparc","SystemZ","X86","XCore"];
-		} else static if (LLVM_Version >= asVersion(3, 5, 0)) {
-			return ["AArch64","ARM","CppBackend","Hexagon","MSP430","Mips","NVPTX","PowerPC","R600","Sparc","SystemZ","X86","XCore"];
-		} else static if (LLVM_Version >= asVersion(3, 4, 0)) {
-			return ["AArch64","ARM","CppBackend","Hexagon","MSP430","Mips","NVPTX","PowerPC","R600","Sparc","SystemZ","X86","XCore"];
-		} else static if (LLVM_Version >= asVersion(3, 3, 0)) {
-			return ["AArch64","ARM","CppBackend","Hexagon","MBlaze","MSP430","Mips","NVPTX","PowerPC","R600","Sparc","SystemZ","X86","XCore"];
-		} else static if (LLVM_Version >= asVersion(3, 2, 0)) {
-			return ["ARM","CellSPU","CppBackend","Hexagon","MBlaze","MSP430","Mips","NVPTX","PTX","PowerPC","Sparc","X86","XCore"];
-		} else {
-			return ["ARM","CellSPU","CppBackend","Hexagon","MBlaze","MSP430","Mips","PTX","PowerPC","Sparc","X86","XCore"];
-		}
-	}().map!(t => "version (LLVM_Target_" ~ t ~ ") targets ~= \"" ~ t ~ "\";").joiner.array);
+			static if (LLVM_Version >= asVersion(4, 0, 0)) {
+				return ["AArch64","AMDGPU","ARM","AVR","BPF","Hexagon","Lanai","Mips","MSP430","NVPTX","PowerPC","RISCV","Sparc","SystemZ","WebAssembly","X86","XCore"];
```

Comment:
What irked me was that there was a change from 3.9 to 4.0 :)
I have no issue with it being sorted alphabetically, but you'll have to do that for all of them then. I need consistency.